modules/ossm-release-notes-extensibility-features.adoc

```diff
@@ -0,0 +1,25 @@
+////
+Module included in the following assemblies:
+* service-mesh-docs-main/ossm-release-notes-support-tables-assembly.adoc
+////
+
+//Post GA: might be worth considering putting tables into their own modules. Might easier to maintain. Warrants further discussion post GA.
+
+:_mod-docs-content-type: REFERENCE
+[id="extensibility-features_{context}"]
+= Extensibility features
+
+[cols="1,1"]
+|===
+| Feature | Status
+
+| WebAssembly Extension
```

Comment:
Just checking if this should be written as "Web Assembly" like the number 1 callout writes it. And extension lowercase?